### PR TITLE
[4.0] Use the new hasField()

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #21445 .

### Summary of Changes
change from property_exists() to the new hasField()method

### CODE REVIEW

**Comment by ghost** [comment]
can #21445 be closed?

**Comment by alikon** [comment]
please hold on, let me re-check first, if i've still missed some places

**Comment by alikon** [comment]
@franz-wohlkoenig re-check done you can close #21445  for me

File: administrator/components/com_banners/Table/ClientTable.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ public function publish($pks = null, $state = 1, $userId = 0)
 		$where = $k . '=' . implode(' OR ' . $k . '=', $pks);
 
 		// Determine if there is checkin support for the table.
-		if (!property_exists($this, 'checked_out') || !property_exists($this, 'checked_out_time'))
+		if (!$this->hasField('checked_out') || !$this->hasField( 'checked_out_time'))
```
Comment (Quy) [inline administrator/components/com_banners/Table/ClientTable.php]: Remove space before `'checked_out_time'`

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7aea8ec6e06369d40d0405ce162177da0b39e591<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21496">issues.joomla.org/tracker/joomla-cms/21496</a>.</sub>

**Pull request merged**